File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-participants_param.ts


import { ISearchParticipants_Param } from './isearch-participants_param';

export class SearchParticipants_Param implements ISearchParticipants_Param {
  
	public SearchString: string = "";
	public Context: string = "";
	public CountryID?: number;
	public TrainingEventID?: number;
	public IncludeVettingOnly?: boolean;
  
}

File: GTTS/INL.SearchService/Models/SearchParticipants_Param.cs

namespace INL.SearchService.Models
{
    public class SearchParticipants_Param : ISearchParticipants_Param
    {
        public string SearchString { get; set; }
        public string Context { get; set; }
        public int? CountryID { get; set; }
        public long? TrainingEventID { get; set; }
        public bool? IncludeVettingOnly { get; set; }
    }
}
